sonaranalyzer-dotnet/its/SonarAnalyzer.Testing.ImportBefore.targets

```diff
@@ -3,14 +3,21 @@
           AfterTargets="ResolveCodeAnalysisRuleSet"
           BeforeTargets="CoreCompile">
 
+    <!-- Copy project-specific SonarLint.xml to enable project-specific settings, such as file headers -->
+    <PropertyGroup>
+      <ProjectSpecificSonarLintXml Condition="Exists('$(MSBuildStartupDirectory)\config\$(PROJECT)\SonarLint.xml')">
+        $(MSBuildStartupDirectory)\config\$(PROJECT)\SonarLint.xml
+      </ProjectSpecificSonarLintXml>
+      <ProjectSpecificSonarLintXml Condition="!Exists('$(MSBuildStartupDirectory)\config\$(PROJECT)\SonarLint.xml')">

```

Comment:
Minor - not worth changing, but this condition could just be   ...='' . It makes it a bit more obvious that it is setting a default/fallback value, and it is also easier to maintain in the future if the condition ever changes.
The other pattern setting a default is to unconditionally set the default value first, then follow it with a property definition with a condition that will overwrite it.